src/routes/routeConstants.ts

```diff
@@ -13,41 +18,55 @@ export const COLONY_EXTENSIONS_ROUTE = `extensions`;
 export const COLONY_INTEGRATIONS_ROUTE = `integrations`;
 export const COLONY_INCORPORATION_ROUTE = `incorporation`;
```

Comment:
We should disable now, in this PR, all the routes that aren't available for launch

Reply:
Can we keep them enabled in dev?